src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -45,13 +45,14 @@
 		<AppSettingsSection v-if="!isGuest"
 			:title="t('spreed', 'Privacy')"
 			class="app-settings-section">
-			<input id="read_status_privacy"
+			<CheckboxRadioSwitch
+				id="read_status_privacy"
 				:checked="readStatusPrivacyIsPublic"
 				:disabled="privacyLoading"
 				type="checkbox"
 				name="read_status_privacy"
 				class="checkbox"
-				@change="toggleReadStatusPrivacy">
+				@update="toggleReadStatusPrivacy" />
 			<label for="read_status_privacy">{{ t('spreed', 'Share my read-status and show the read-status of others') }}</label>
```

Comment:
```suggestion
				@update:checked="toggleReadStatusPrivacy">
				{{ t('spreed', 'Share my read-status and show the read-status of others') }}
			</CheckboxRadioSwitch>
```

Otherwise:
1. The label is misplaced (below the checkbox)
2. Saving the value does not work

Reply:
this is still broken?!